osi_sensorview.proto

```diff
@@ -173,63 +177,53 @@ message RadarSensorView
     //
     optional RadarSensorViewConfiguration view_configuration = 1;
 
-    // Ray tracing data.
+    // Received signal.
+    //
+    // This field includes the returned signal in the scene, that is backscattered to the sensor.
     //
-    // This field includes one entry for each ray, in left-to-right,
-    // top-to-bottom order (think of scan lines in a TV).
+    // \note OSI uses singular instead of plural for repeated field names.
     //
-    repeated Reflection reflection = 2;
+    repeated ReceivedSignalElement received_signal_element = 2;
 
     //
-    // \brief Definition of the radar reflection.
+    // \brief Definition of the result of a e.g. rendering algorithm, which simulates the
+    // propagation of signal in a virtual scene. 
     //
-    message Reflection
+    message ReceivedSignalElement
     {
-        // Relative signal level of the reflection.
+        // Received spatial element.
         //
-        // This takes the combined antenna diagram (losses in TX and RX)
-        // as well as the signal losses due to scattering and absorption
-        // into account, and will, when multiplied by TX power yield the
-        // actual RX power.
+        // This describes the incoming direction and signal strength in dB
+        // of the signal return from the simulated scene.
+        // The signal can be received from a different angle than it has been emitted to.
         //
-        // Unit: dB
+        // \note Data is in sensor coordinate system.
         //
-        optional double signal_strength = 1;
-
+        optional SpatialSignalGain spatial_signal_gain = 1;
+        
         // Time of flight.
         //
-        // This is the time of flight of the reflection, which is directly
-        // proportional to the distance traveled.
+        // This is the time needed by the signal from emission to reception.
         //
         // Unit: s
         //
         optional double time_of_flight = 2;
 
-        // Doppler shift.
+        // Doppler shift
         //
         // Shift in frequency based on the specified TX frequency.
         //
         // Unit: Hz
         //
         optional double doppler_shift = 3;
 
-        // TX horizontal angle (azimuth).
-        //
-        // Horizontal angle of incidence of the source of the reflection
-        // at the TX antenna.
-        //
-        // Unit: rad
-        //
-        optional double source_horizontal_angle = 4;
-
-        // TX vertical angle (elevation).
-        //
-        // Vertical angle of incidence of the source of the reflection
-        // at the TX antenna.
+        // The phase of the spatial_signal_gain at the sensor.
         //
+        // The phase_size() has to be equal to the spatial_signal_gain.signal_gain_size().
```

Comment:
Second option is implemented because of usage of `SpatialSignalGain` or `SpatialSignalStrength` in other messages in OSI like `EmittingStructureAttribute`